dll/win32/shell32/dialogs/drive.cpp

```diff
@@ -230,54 +231,24 @@ InsertDefaultClusterSizeForFs(HWND hwndDlg, PFORMAT_DRIVE_CONTEXT pContext)
 
     if (!wcsicmp(wszBuf, L"FAT16") ||
         !wcsicmp(wszBuf, L"FAT")) //REACTOS HACK
-    {
```

Comment:
Since your if conditions take more than one line, and there are in if... else if ... pattern, please keep the braces for each case.